Alkitab/src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,7 @@
 
 		<activity
 			android:name="yuku.alkitab.base.IsiActivity"
+			android:configChanges="colorMode|uiMode"
```

Comment:
Hi, thanks for your contribution! May I ask why `colorMode` is included?

Reply:
Uh, I believe that is an error, and I just misunderstood what the difference is.